client/src/components/profile/owners-restaurants.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from 'react';
```

Comment:
Could you remove background image from this page?
![image](https://user-images.githubusercontent.com/6152066/80944387-20cb0700-8df2-11ea-82ba-6b752b2b57f0.png)

Reply:
Don't show pages if restaurant list is empty
![image](https://user-images.githubusercontent.com/6152066/80944517-756e8200-8df2-11ea-8a80-4e7112ea8d6e.png)